src/extension.ts

```diff
@@ -25,8 +25,8 @@ import * as httpgdViewer from './plotViewer';
 import { RMarkdownPreviewManager } from './rmarkdown/preview';
 
 // global objects used in other files
-export const extDir: string = path.join(os.homedir(), '.vscode-R');
-export const tmpDir: string = path.join(extDir, 'tmp');
+export const extensionDirectory: string = path.join(os.homedir(), '.vscode-R');
```

Comment:
I think both `extDir` or `extensionDirectory` are ambiguous because it could mean the installation directory, but I don't come up with a name that makes more sense. Any ideas?

Reply:
I agree. Hmm, what about... localDirectory? homeDirectory?

Reply:
Given it's created in the ~/ directory, I think something relating to 'home' is closer to the right name.

Just throwing ideas out there, because honestly I'm not sure either:
- homeDir
- homeExtDir
- extHomeDir

Reply:
I like `homeExtDir`, which is almost the same as its path.